### PR TITLE
Update css extract plugin parameters in prod webpack configuration

### PR DESCRIPTION
**Pull request by efredine**

### CODE REVIEW

File: webpack.production.config.js [hunk 1]
```diff
@@ -81,7 +81,7 @@ module.exports = {
             test: /\.scss$/,
             // we extract the styles into their own .css file instead of having
             // them inside the js.
-            loader: ExtractTextPlugin.extract('style', 'style!css?modules&localIdentName=[name]---[local]---[hash:base64:5]!sass')
+            loader: ExtractTextPlugin.extract('style', 'css?modules&localIdentName=[name]---[local]---[hash:base64:5]!sass')
```
Comment (ambrons) [inline webpack.production.config.js:84]: I second, that this is the right fix for using with ExtractTextPlugin.

**Comment by jpsierens** [comment]
Thanks a lot for this!

**Comment by efredine** [comment]
You're welcome.  Glad it did the right thing.  You have the dubious honour of being the recipient of my first pull request.  (I'm a long-time management type who has decided to get back into coding - and loving it.)

**Comment by jpsierens** [comment]
Thats really nice to hear! I myself have a business degree but turned web developer almost right after. There's no comparison in the levels of satisfaction that I get from programming.

**Pull request merged**